Makefile

```diff
@@ -66,7 +66,7 @@ export PATH := $(CURDIR)/cmd/bin:$(PATH)
 
 # Build subctl before deploying to ensure we use that
 # (with the PATH set above)
-deploy: cmd/bin/subctl
+deploy: cmd/bin/subctl images
```

Comment:
This shouldn't be necessary, why did you add it?

```suggestion
deploy: cmd/bin/subctl
```

Reply:
The image is always built before deployment, no need to add this.
The problem you had was probably something else (perhaps you had stale shipyard image/files around).